### PR TITLE
Add theme for Visual Studio Code

### PR DESCRIPTION
**Pull request by tomaszwojcik**
Just copy `Spacedust` from `Visual Studio Code` to `~/.vscode/extensions`.

Theme is just copy of TextMate theme with slight fixes.

### CODE REVIEW

**Comment by tomaszwojcik** [comment]
Should I update readme as well?

**Comment by cschep** [comment]
Thank you for porting this!

**Comment by chow-land** [comment]
thanks for this!

File: Visual Studio Code/Spacedust/themes/Spacedust.tmTheme [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+<dict>
+	<key>name</key>
+	<string>Spacedust</string>
+	<key>settings</key>
+	<array>
+		<dict>
+			<key>settings</key>
+			<dict>
+				<key>background</key>
+				<string>#0A171B</string>
```
Comment (hallski) [inline Visual Studio Code/Spacedust/themes/Spacedust.tmTheme:13]:
```suggestion
				<string>#041E23</string>
```

**Comment by hallski** [comment]
First of all, terribly sorry for the lack of response. For some reason I completely missed that there was a PR open.

File: Visual Studio Code/Spacedust/themes/Spacedust.tmTheme [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+		</dict>
+		<dict>
+			<key>name</key>
+			<string>User-defined constant</string>
+			<key>scope</key>
+			<string>constant.character, constant.other</string>
+			<key>settings</key>
+			<dict>
+				<key>foreground</key>
+				<string>#4A9D8F</string>
```
Comment (hallski) [inline Visual Studio Code/Spacedust/themes/Spacedust.tmTheme:80]:
```suggestion
				<string>#CA763C</string>
```

File: Visual Studio Code/Spacedust/themes/Spacedust.tmTheme [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+	<string>Spacedust</string>
+	<key>settings</key>
+	<array>
+		<dict>
+			<key>settings</key>
+			<dict>
+				<key>background</key>
+				<string>#0A171B</string>
+				<key>caret</key>
+				<string>#FF8D3D</string>
```
Comment (hallski) [inline Visual Studio Code/Spacedust/themes/Spacedust.tmTheme:15]:
```suggestion
				<string>#ECF0C1</string>
```

File: Visual Studio Code/Spacedust/themes/Spacedust.tmTheme [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+		</dict>
+		<dict>
+			<key>name</key>
+			<string>Function argument</string>
+			<key>scope</key>
+			<string>variable.parameter</string>
+			<key>settings</key>
+			<dict>
+				<key>foreground</key>
+				<string>#305F65</string>
```
Comment (hallski) [inline Visual Studio Code/Spacedust/themes/Spacedust.tmTheme:161]:
```suggestion
				<string>#CA763C</string>
```

File: Visual Studio Code/Spacedust/themes/Spacedust.tmTheme [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+		</dict>
+		<dict>
+			<key>name</key>
+			<string>Tag name</string>
+			<key>scope</key>
+			<string>entity.name.tag</string>
+			<key>settings</key>
+			<dict>
+				<key>foreground</key>
+				<string>#EBC562</string>
```
Comment (hallski) [inline Visual Studio Code/Spacedust/themes/Spacedust.tmTheme:172]:
```suggestion
				<string>#E35B00</string>
```

File: Visual Studio Code/Spacedust/themes/Spacedust.tmTheme [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+		</dict>
+		<dict>
+			<key>name</key>
+			<string>Tag attribute</string>
+			<key>scope</key>
+			<string>entity.other.attribute-name</string>
+			<key>settings</key>
+			<dict>
+				<key>foreground</key>
+				<string>#305F65</string>
```
Comment (hallski) [inline Visual Studio Code/Spacedust/themes/Spacedust.tmTheme:183]:
```suggestion
				<string>#09699D</string>
```

**Review by hallski — CHANGES_REQUESTED** [review]
🎉 

Thanks a lot for the PR, I actually started porting this to VSCode as well at some point but never finished. I made some suggestions where there were some deviations from the theme. 

Would really need to give it a touch up at some point though and possibly add some more colors to tackle an editor like VSCode as well as the languages of today.

Not sure if you are still watching this but will give you some time to respond and will otherwise merge it with the suggested changes.

**Comment by hallski** [comment]
Oh and yes please do add yourself to the README if you like :)

**Comment by tomaszwojcik** [comment]
Hi! Applied requested changes (manually, not on GH) and it is ready to be merged :)

**Review by hallski — APPROVED** [review]
Awesome, and thanks a lot for your patiens with me completely missing the PR!

**Pull request merged**